pkgs/applications/office/qownnotes/default.nix

```diff
@@ -1,29 +1,69 @@
-{ mkDerivation, lib, stdenv, fetchurl
-, qmake, qttools, qtbase, qtsvg, qtdeclarative, qtxmlpatterns, qtwebsockets
-, qtx11extras, qtwayland
+{ lib
+, stdenv
+, fetchurl
+, qmake
+, qttools
+, qtbase
+, qtdeclarative
+, qtsvg
+, qtwayland
+, qtwebsockets
+, qtx11extras
+, qtxmlpatterns
+, makeWrapper
+, wrapQtAppsHook
```

Comment:
The reformatting is unnecessary.

Reply:
No, it isn't. According to a recent review by @fabaff, this is preferred going forward, as it leads to cleaner Git diffs, so it is apparent at first glance, which dependencies were added/removed on package updates.

Reply:
I would prefer the reformatting to be in a separate commit to make the diff of **this** single-commit PR cleaner. But anyway, I guess I'm okay with that.

Reply:
I put it in a separate commit now